src/test/java/com/jcabi/github/mock/MkHookTest.java

```diff
@@ -107,26 +139,28 @@ public void createWithCorrectRepo() throws Exception {
      * Test if {@link MkHook} is being created with the correct id.
      * @throws Exception If something goes wrong
      */
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     public void createWithCorrectId() throws Exception {
         MatcherAssert.assertThat(
             "Hook json returned wrong id",
             this.hook.json().getInt("id"),
-            new IsEqual<Integer>(Tv.FIVE)
+            new IsEqual<Integer>(
+                Integer.parseInt(this.xml.xpath("id/text()").get(0))
```

Comment:
@llorllale I think that we should test real values here; we are testing if we can get `id` correctly from json, and albeit this implementation works, I think that we should use the real value here instead looking for it in the xml